js/shared/src/logistics/redux/selectors.js

```diff
@@ -133,6 +133,10 @@ export const selectAllTours = createSelector(
   }
 )
 
+export const selectTourById = (state, tourId) => {
+  return selectAllTours(state).find(t => t['@id'] === tourId)
+}
+
```

Comment:
OK why not I change

Just so I know what would you do that? it seems to me that anyway a selector is no use here, because `tourId` arg will change then there is no memoization?

Reply:
yes but I dont think this really work, as `tourId` is basically changing all the time? (ref : [https://redux.js.org/usage/deriving-data-selectors#createselector-behavior](https://redux.js.org/usage/deriving-data-selectors#createselector-behavior))